cmd/lifecycle/restorer.go

```diff
@@ -50,6 +50,7 @@ func (r *restoreCmd) DefineFlags() {
 		cli.FlagBuildImage(&r.BuildImageRef)
 	}
 	cli.FlagAnalyzedPath(&r.AnalyzedPath)
+	cli.FlagRunPath(&r.RunPath)
```

Comment:
I am sorry to raise the specter of another Platform API version bump, but I think we need to gate this on Platform 0.14 also (an raise another spec PR).